ui/widgets/nuxeo-user-avatar.js

```diff
@@ -299,33 +307,40 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
           let tempName = '';
           const splitName = name.split(' ');
           const alphabetPosition = alphabet.indexOf(name.charAt(0).toLowerCase());
```

Comment:
nit: despite being correct (and we can discuss if helps readability, maybe it helps!), we don't need that variable.

Reply:
Since there is some ongoing discussions about a major refactor on this code, I would suggest to leave it as it is, even though I agree this could be discussed.